test/Microsoft.Azure.Devices.Edge.Test/helpers/Context.cs

```diff
@@ -91,7 +91,8 @@ IEnumerable<Registry> GetAndValidateRegistries()
             this.MethodSenderImage = Option.Maybe(Get("methodSenderImage"));
             this.OptimizeForPerformance = context.GetValue("optimizeForPerformance", true);
             this.PackagePath = Option.Maybe(Get("packagePath"));
-            this.Proxy = Option.Maybe(context.GetValue<Uri>("proxy"));
+            this.AgentProxy = Option.Maybe(context.GetValue<Uri>("agentProxy"));
```

Comment:
>AgentProxy [](start = 17, length = 10)

maybe TestRunnerProxy? I stared at this for a good 30sec trying to understand why the EdgeAgent needed a different proxy :D

Reply:
Fixed